apiary-metastore-events/apiary-hive-events/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,4 @@
+log4j.rootLogger=INFO, A1
```

Comment:
We've standardised on using log4j.xml in our other projects as you get validation on the config. Please replace this.

Reply:
moved all of these to log4j.xml and added some additional exclusions to clean up some bind warnings I had missed